File: result/translate/transcoder/python2java/java/FIND_THE_MINIMUM_DISTANCE_BETWEEN_TWO_NUMBERS.java
public static int f_gold ( int [ ] arr , int n , int x , int y ) {
  int minDist = 2147483647 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    for ( int j = i + 1 ;
    j < n ;
    j ++ ) {
      if ( ( x == arr [ i ] && y == arr [ j ] || y == arr [ i ] && x == arr [ j ] ) && minDist > Math . abs ( i - j ) ) {
        minDist = Math . abs ( i - j ) ;
      }
    }
  }
  return minDist ;
}

File: result/translate/starchat/java2python/python/FIND_THE_MINIMUM_DISTANCE_BETWEEN_TWO_NUMBERS.py
def f_gold(arr, n, x, y):
        min_dist = float('inf')
        for i in range(n):
            for j in range(i + 1, n):
                if (x == arr[i] and y == arr[j]) or (y == arr[i] and x == arr[j]):
                    min_dist = min(min_dist, abs(i - j))
        return min_dist